### PR TITLE
Vsfp 28 document pros cons of classical web app

### PR DESCRIPTION
**Pull request by lstreckeisen**

### CODE REVIEW

File: SA/design_concept/content/poc/web_app.typ [hunk 1]
```diff
@@ -0,0 +1,12 @@
+= Web Application
+A web application has a frontend and a backend.
+The frontend is served to the user in a web browser and communicates with the backend, which is hosted on a server, via a REST API.
```
Comment (fl3pp) [inline SA/design_concept/content/poc/web_app.typ:3]:
```suggestion
The frontend is served to the user statically in a web browser and communicates with the backend, which is hosted on a server, via an API.
```

File: SA/design_concept/content/poc/web_app.typ [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Depending on the expected load generated by the frontend, the backend can be horizontally scaled, in which case that frontend communicates with the backend via a load balancer.
```
Comment (fl3pp) [inline SA/design_concept/content/poc/web_app.typ:4]:
```suggestion
```

Do we need this line?

File: SA/design_concept/content/poc/web_app.typ [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+Pros:
+- Frontend and backend can be written in completely different programming languages
+- JavaScript has a wide range of libraries available for frontend development
+
+Cons:
+- JavaScript has a weak type-system
+- The backend would have to organize VisualFP project files for multiple users on a server
```
Comment (fl3pp) [inline SA/design_concept/content/poc/web_app.typ:12]:
```suggestion
- The backend would have to organize VisualFP project files for multiple users on a server
- A hosting solution is required to run the application
```

This was the comment from Farhad

File: SA/design_concept/content/poc/web_app.typ (hunk 2 above)
Comment (lstreckeisen) [reply SA/design_concept/content/poc/web_app.typ:4]: Not really, I just ran out of things to write 😆

Comment (lstreckeisen) [reply SA/design_concept/content/poc/web_app.typ]: I took it out

**Pull request merged**